acrobat/blocks/eventwrapper/eventwrapper.js

```diff
@@ -92,6 +93,7 @@ export default function init(element) {
         break;
       case UPLOAD_START:
         setCurrentEvent('upload');
+        if (reviewBlock[0]) { reviewBlock[0].classList.add('fade'); };
```

Comment:
you might want to hide it immediately on upload (simultaneously or prior to shifting the content upward) then fade it back in after the upload completes?  It might even be possible to do this with only CSS rules